scripts/Util.py

```diff
@@ -3436,25 +3433,25 @@ class Config(Mapping.Config):
 
         @classmethod
         def getSupportedArgs(self):
-            return ("", ["dotnetcore", "framework="])
+            return ("", ["dotnet", "framework="])
 
         @classmethod
         def usage(self):
             print("")
             print("C# mapping options:")
-            print("--dotnetcore                    Run C# tests using .NET Core")
+            print("--dotnet                        Run C# tests using .NET instead of .NET Framework")
```

Comment:
I find these options puzzling. I think it would be clearer to have a single list and use the TFM as identifier:
https://learn.microsoft.com/en-us/dotnet/standard/frameworks
```
--framework=net45|net6.0|net7.0
```

Reply:
Fixed as suggested.